components/home-page/users/Users.js

```diff
@@ -59,7 +59,7 @@ export default function Users(props) {
         <Row>
           <Col sm={18} md={12} className={styles.logo}>
               <a href={`${prefix}/case-studies/ballerina-central`}><img src={`${prefix}/images/home-page/user-logos/ballerina-central.svg`} width={340} alt="Ballerina Central" title="Ballerina Central" /></a>
-              <a href="https://www.redcross.lk/news/sri-lanka-red-cross-societys-elixir-a-source-of-strength-to-the-health-sector-of-sri-lanka/" target='_blank' rel="noreferrer"><img src={`${prefix}/images/home-page/user-logos/redcross-elixir.png`} width={280} alt="Redcross Elixir" title="Redcross Elixir" /></a>
+              <a href={`${prefix}/case-studies/redcross-elixir`} target='_blank' rel="noreferrer"><img src={`${prefix}/images/home-page/user-logos/redcross-elixir.png`} width={280} alt="Redcross Elixir" title="Redcross Elixir" /></a>
```

Comment:
Do we need to open this on a new tab since this is on the same site?
```suggestion
              <a href={`${prefix}/case-studies/redcross-elixir`} rel="noreferrer"><img src={`${prefix}/images/home-page/user-logos/redcross-elixir.png`} width={280} alt="Redcross Elixir" title="Redcross Elixir" /></a>
```

Reply:
`rel="noreferrer"` is aso not needed.

Reply:
Removed.